src/Symfony/Component/Validator/Tests/Fixtures/Entity.php

```diff
@@ -19,7 +19,7 @@
  * @Assert\GroupSequence({"Foo", "Entity"})
  * @Assert\Callback({"Symfony\Component\Validator\Tests\Fixtures\CallbackClass", "callback"})
  */
-class Entity extends EntityParent
+class Entity extends EntityParent implements EntityInterface, EntityParentInterface
```

Comment:
Yes... I guess I see your point. Although the fix is going to take care of this case too.

I also added the interface on the `EntityParent` this case (interface implemented through class inheritance) was already taken care of in the code before my patch, but not in the test. I'll add (yet) another interface to take care of this.